adminer/lang/pt-br.inc.php

```diff
@@ -66,19 +66,19 @@
 	'Foreign keys' => 'Chaves estrangeiras',
 	'Triggers' => 'Triggers',
 	'View' => 'Visão',
-	'Unable to select the table' => 'Não é possivel selecionar a Tabela',
-	'Invalid CSRF token. Send the form again.' => 'Token CSRF inválido. Enviar o formulario novamente.',
+	'Unable to select the table' => 'Não é possível selecionar a Tabela',
+	'Invalid CSRF token. Send the form again.' => 'Token CSRF inválido. Enviar o formulário novamente.',
 	'Comment' => 'Comentário',
 	'Default values' => 'Valores padrões',
 	'%d byte(s)' => array('%d byte', '%d bytes'),
 	'No commands to execute.' => 'Nenhum comando para executar.',
 	'Unable to upload a file.' => 'Não é possível enviar o arquivo.',
 	'File upload' => 'Importar arquivo',
-	'File uploads are disabled.' => 'Importação de arquivos desablilitada.',
+	'File uploads are disabled.' => 'Importação de arquivos desabilitada.',
```

Comment:
```suggestion
	'File uploads are disabled.' => 'Importação de arquivos estão desligados.',
```

Reply:
'Importação de arquivos estão **desligados**.' It is wrong for brazilian portuguese speakers.
alternatives:
> Importação de arquivos **está desligada**

> Importação de arquivos **desablilitada**  (it's is my choise)

Reply:
Well, **está desligada** also make sense.

Meanwhile the word **desablilitada** can be substituted with **desactivada.** Which is more generic, IMO.

Reply:
word **desactivada** don't exists in brazilian portuguese.

Reply:
1. https://www.dicio.com.br/desativada/
2. https://www.dicionarioinformal.com.br/desativado/